examples/47-pixelformats/pixelformats.cpp

```diff
@@ -0,0 +1,421 @@
+/*
+ * Copyright 2011-2022 Branimir Karadzic. All rights reserved.
```

Comment:
I would encourage you to add your name here, since you created the example. If you don't wish it's fine as-is.